streaming/multimodal/convert/laion/laion400m/convert_and_upload.py

```diff
@@ -220,6 +221,9 @@ def convert_and_upload_shards(args: Namespace) -> bool:
             print(f'Shard {idx:05}: converting...')
             convert(parquet_filename, mds_dirname, hashes)
         print(f'Shard {idx:05}: uploading...')
+        rename_mds_shard_filename = os.path.join(mds_dirname, f'shard.{idx:05}.mds')
```

Comment:
Sorry, what/why is happening in these three lines? Could use a comment

Reply:
Added a comment in https://github.com/mosaicml/streaming/pull/220/commits/626aa6be526e4e7e11152926881915bb62a9182d.